resources/README.md

```diff
@@ -0,0 +1,45 @@
+### [Official Website](https://webassembly.org)
+
+## Documentation
+
+#### Specifications
+ - [Core on webpage](https://webassembly.github.io/spec/core)
+ - [Core on PDF](https://webassembly.github.io/spec/core/_download/WebAssembly.pdf)
+
+#### Proposals
+ - [Proposals Roadmap](https://webassembly.org/roadmap)
+ - [Proposals repo on Github](https://github.com/WebAssembly/proposals)
+
+#### Other
+ - [MDN WebAssembly Documentation](https://developer.mozilla.org/en-US/docs/WebAssembly)
+ - [Design doc on Github](https://github.com/WebAssembly/design)
+ - [Interactive Opcode Table](https://pengowray.github.io/wasm-ops)
+ - [Mechanising and Verifying the WebAssembly Specification (PDF)](https://www.cl.cam.ac.uk/~caw77/papers/mechanising-and-verifying-the-webassembly-specification.pdf)
+
+## Articles
+ - [A cartoon intro to WebAssembly by Lin Clark](https://hacks.mozilla.org/2017/02/a-cartoon-intro-to-webassembly)
+
+## Made with WebAssembly
+ - Figma
+ [(1)](https://www.figma.com/blog/webassembly-cut-figmas-load-time-by-3x/)
+ [(2)](https://www.figma.com/blog/how-we-built-the-figma-plugin-system)
+ - [Google Earth](https://blog.chromium.org/2019/06/webassembly-brings-google-earth-to-more.html)
+ - [AutoCAD Web App](https://blogs.autodesk.com/autocad/autocad-web-app-google-io-2018)
+ - [1Password](https://blog.1password.com/1password-x-may-2019-update)
+ - [GIVE ME MORE!](https://madewithwebassembly.com)
+
+## Playgrounds
+ - [Webassembly Studio](https://webassembly.studio)
+ - [Wasm Explorer](https://mbebenita.github.io/WasmExplorer)
+
+## Awesome Lists
+
+ - [Awesome Wasm](https://github.com/mbasso/awesome-wasm)<br/>
+ Collection of general awesome things regarding WebAssembly ecosystem.
+ - [Awesome WebAssembly Languages](https://github.com/appcypher/awesome-wasm-langs)<br/>
+ Collection of languages that currently compile to or have their VMs in WebAssembly.
+ - [Awesome WebAssembly Runtimes](https://github.com/appcypher/awesome-wasm-runtimes)<br/>
+ Collection of virtual machines and tools that execute the WebAssembly.
+
+## Community & Social Media
+ - [Community Group](https://www.w3.org/community/webassembly)
```

Comment:
Probably this link is not so helpful. I think Discord chat and Telegram chat links will be more helpful from what people could expect from the header